docusaurus/docs/iOS/introduction.md

```diff
@@ -92,37 +93,40 @@ struct ChannelView: View {
 }
 ```
 
-<p>&nbsp;</p>
+### StreamChatUI Components
 
-## 6. **StreamChatUI** components behave similarly to native UIKit components
+UI SDK components behave similarly to native UIKit components:
 
-**They respect the `tintColor` of their current view hierarchy:**
+#### Components respect the `tintColor` of their current view hierarchy 
 
+<!-- side by side component -->
 | default `tintColor`  | `tintColor = .systemPink` |
 | ------------- | ------------- |
-| ![Chat UI with default tint color](assets/blue-tint.png)  | ![Chat UI with pink tint color](assets/pink-tint.png)  |
+| ![Chat UI with default tint color](../assets/blue-tint.png)  | ![Chat UI with pink tint color](../assets/pink-tint.png)  |
```

Comment:
This will not work because the `assets` directory is situated in the same folder as `introduction.md`. Please change it to `![Chat UI with default tint color](assets/blue-tint.png)`. The same applies to all links that lead one directory above in the hiearchy